### DISCUSSION TITLE
playerId , Ids

### DISCUSSION THREAD

**Discussion by bestdeveloperjo** [naverz/zepeto-studio-kor#963] [5. Q&A]
playerId
zepetoId : 제페토 계정 아이디
userId 

이렇게 알고있습니다.

나머지  playerId,userId는 어떤 용도의 아이디 입니까?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 질문 주신 내용 답변 드립니다.

**UserId** : 제페토 시스템에 가입시 유저에게 발급되는 유니크한 아이디입니다. 항상 동일하며 변경할 수 없습니다.
월드 개발시에는 WorldService를 사용해서 월드에 접속한 유저의 UserId를 받아올 수 있습니다.

**ZepetoID** : 제페토 앱 내에서 보여지는 아이디로, 유저가 직접 생성하고 변경할 수 있는 값입니다.
 
**Session ID** : 멀티플레이 Room에 입장할 때마다, 유저에게 부여되는 고유값입니다.

**Player ID** : Character Controller에서 ZEPETO Player 단위를 구분하는 고유값입니다.
멀티플레이에서 CreatePlayerWithUserId를 통한 ZEPETO Player 생성시, playerId 자리에 sessionId를 넣으면
멀티플레이 세션을 단위로 한 ZEPETO Player가 생성되고, Character Controller 멀티플레이 컨텐츠 제작에 쉽게 활용할 수 있습니다.

감사합니다.
